addon/chrome.manifest

```diff
@@ -36,9 +36,7 @@ locale    tabmixplus   zh-TW         chrome/locale/zh-TW/
 
 skin      tabmixplus   classic/1.0   chrome/skin/
 
-skin      tabmix-version tabmixplus  chrome/skin/app_version/39.0/       appversion>=39.0a1
-skin      tabmix-version tabmixplus  chrome/skin/app_version/60.0/       appversion>=60.0a1
-skin      tabmix-os    classic/1.0   chrome/skin/app_version/41.0/win/   os=WINNT appversion>=41.0a1
-skin      tabmix-os    classic/1.0   chrome/skin/app_version/60.0/win/   os=WINNT appversion>=60.0a1
+skin      tabmix-version tabmixplus  chrome/skin/app_version/78/       appversion>=78
```

Comment:
Why 78?
Is it for Waterfox

Reply:
The `appversion>=xx` means this rule apply to versions above xx.
78 is the last ESR version, and yes WF current is base on that version.
The firefox UI structure stabled at what we see today back to about 73, I think set minimum support version to 78 is reasonable enough.